### PR TITLE
[#IR-265] Issue IDs containing leading/trailing (white)spaces are now trimmed before actually searching them in the board which prevents duplicate entry creation

### PR DESCRIPTION
**Pull request by himanshu-garg-165**

### CODE REVIEW

File: apps/retrospective/models/task.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/iReflect/reflect-app/apps/retrospective"
 	"github.com/iReflect/reflect-app/db/models/fields"
+	"strings"
```
Comment (chetanyakan) [inline apps/retrospective/models/task.go:15]: Imports are organized in groups, with blank lines between them. The standard library packages are always in the first group.

https://github.com/golang/go/wiki/CodeReviewComments#imports

File: apps/retrospective/services/sprint_sync.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/iReflect/reflect-app/libs/utils"
 	"github.com/iReflect/reflect-app/workers"
 	"github.com/jinzhu/gorm"
+	"strings"
```
Comment (chetanyakan) [inline apps/retrospective/services/sprint_sync.go:21]: Same as above. We should try to order imports as:
```go
import (
     "standard library packages"

     "external packages"

     "internal packages"
)
```

https://github.com/golang/go/wiki/CodeReviewComments#imports

File: apps/timetracker/providers.go [hunk 3]
```diff
@@ -14,7 +14,7 @@ type TimeProvider interface {
 
 // Connection ...
 type Connection interface {
-	GetProjectTimeLogs(project string, startTime time.Time, endTime time.Time) []serializers.TimeLog
+	GetProjectTimeLogs(project string, startTime time.Time, endTime time.Time) []*serializers.TimeLog
```
Comment (chetanyakan) [inline apps/timetracker/providers.go:17]: When two or more consecutive named function parameters share a type, you can omit the type from all but the last. In this case, we can write:

```go
GetProjectTimeLogs(project string, startTime, endTime time.Time) []*serializers.TimeLog
```

https://tour.golang.org/basics/5

File: apps/timetracker/providers.go [hunk 4]
```diff
@@ -34,7 +34,7 @@ func GetTimeProvider(name string) TimeProvider {
 }
 
 // GetProjectTimeLogs ...
-func GetProjectTimeLogs(config []byte, project string, startTime time.Time, endTime time.Time) (timeLogs []serializers.TimeLog, err error) {
+func GetProjectTimeLogs(config []byte, project string, startTime time.Time, endTime time.Time) (timeLogs []*serializers.TimeLog, err error) {
```
Comment (chetanyakan) [inline apps/timetracker/providers.go:37]: Same as above. We can have: 
```go
func GetProjectTimeLogs(config []byte, project string, startTime, endTime time.Time) (timeLogs []*serializers.TimeLog, err error) {
```

https://tour.golang.org/basics/5

File: apps/timetracker/providers/gsheet.go [hunk 5]
```diff
@@ -89,9 +89,9 @@ func getConfigObject(config interface{}) (GsheetConfig, error) {
 }
 
 // GetProjectTimeLogs ...
-func (m *GsheetConnection) GetProjectTimeLogs(project string, startTime time.Time, endTime time.Time) []serializers.TimeLog {
+func (m *GsheetConnection) GetProjectTimeLogs(project string, startTime time.Time, endTime time.Time) []*serializers.TimeLog {
```
Comment (chetanyakan) [inline apps/timetracker/providers/gsheet.go:92]: Same as above. We can have:

```go
func (m *GsheetConnection) GetProjectTimeLogs(project string, startTime, endTime time.Time) []*serializers.TimeLog {
```

https://tour.golang.org/basics/5

**Comment by anmolghosh** [comment]
Closing this as like we discussed with @ayadav, this things should not be handled on ireflect's end.
Go through the comments though.

**Pull request closed without merging**